src/map/status.hpp

```diff
@@ -3619,6 +3619,7 @@ status_change *status_get_sc(struct block_list *bl);
 
 bool status_isdead(block_list &bl);
 int32 status_isimmune(struct block_list *bl);
+bool status_isendure(block_list &bl, t_tick tick);
```

Comment:
```suggestion
bool status_isendure(block_list& bl, t_tick tick);
```

Reply:
Saw that the other status_is function use it like that and thought it looks better. You prefer it next to the type rather than the object name?